Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml

```diff
@@ -1307,8 +1299,33 @@ tasks:
           accessor: agentid
       FilePath:
         complex:
-          root: foundIncidents.CustomFields
-          accessor: initiatorpath
+          root: alert
+          transformers:
+          - operator: If-Then-Else
+            args:
+              condition:
+                value:
+                  simple: lhs!=rhs
+              conditionB: {}
+              conditionInBetween: {}
+              else:
+                value:
+                  simple: foundIncidents.CustomFields.initiatorpath
+                iscontext: true
+              equals: {}
+              lhs:
+                value:
+                  simple: alert.filesha256
```

Comment:
note that you are setting a path based on an SHA key -> hope that this is intentional.